samigo/samigo-app/src/webapp/js/authoring.js

```diff
@@ -612,9 +602,10 @@ function initTimedRadio(){
 }
 
 function initAnononymousUsers(){
-	var releaseToVal = $('#assessmentSettingsAction\\:releaseTo').val();
-	if (releaseToVal === 'Anonymous Users') {
-		handleAnonymousUsers(releaseToVal, "");
+	var releaseTo = document.getElementById('assessmentSettingsAction:releaseTo');
+	releaseTo.prevValue = releaseTo.value;
+	if (releaseTo.value === 'Anonymous Users') {
```

Comment:
Will this hardcoded English work when the user's locale is not English?

Reply:
I was wondering about that too at first. But it refers to the value of the select, not to the label, which is independent of the users language settings.

Reply:
Good stuff, thanks for the clarification!